### DISCUSSION TITLE
Should we have tests for tutorial notebooks?

### DISCUSSION THREAD

**Discussion by bobturneruk** [pykale/pykale#204] [Q&A] [answered]
In the first instance, these could just check that the notebooks run to completion.

This may be useful (https://github.com/marketplace/actions/run-notebook), but it might be easier just to script it i.e. `jupyter nbconvert --execute <notebook>`.

**Comment by haipinglu** [comment, marked as answer]
If yes, will this be under pykale/tests/ or workflow/actions?

Our tests have been focusing on the core API. We did not test examples yet (belonging to regression tests that we did not start working on yet). 

The simple "test" may be incorporated considering its "low cost".

**Reply by bobturneruk** [reply]
This is now covered by #214
